src/webgpu/api/validation/render_pipeline/float32_blendable.spec.ts

```diff
@@ -0,0 +1,46 @@
+export const description = `
+Tests for capabilities added by float32-blendable flag.
+`;
+
+import { makeTestGroup } from '../../../../common/framework/test_group.js';
+import { ColorTextureFormat } from '../../../format_info.js';
+
+import { CreateRenderPipelineValidationTest } from './common.js';
+
+export const g = makeTestGroup(CreateRenderPipelineValidationTest);
+
+const kFloat32Formats: ColorTextureFormat[] = ['r32float', 'rg32float', 'rgba32float'];
+
+g.test('create_render_pipeline')
+  .desc(
+    `
+Tests that the float32-blendable feature is required if blend is not undefined.
```

Comment:
It's unclear this refers to the render pipeline descriptor. A bit clearer:
```suggestion
Tests that the float32-blendable feature is required to create a render
pipeline that uses blending with any float32-format attachment.
```